servers.controllers.js

```diff
@@ -7,3 +11,24 @@ exports.getTopics = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.getByEndpoints = (req, res, next) => {
+  getDocsByEndpoints()
+    .then((endpoints) => {
+      res.status(200).send({ endpoints });
+    })
+    .catch(next);
+};
+
+exports.getByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+
+  getObjByArticeleId(article_id)
+    .then((article) => {
+      if (!article) {
+        return Promise.reject({ status: 404, msg: "Not Found" });
```

Comment:
Rejected Promises should be handled in the Model, not the controller.

Reply:
Hi, revised accordingly